### DISCUSSION TITLE
Smokeping Upgrade - Version 2.6.11 to 2.8.2

### DISCUSSION THREAD

**Discussion by usaiat** [oetiker/SmokePing#345] [Q&A]
I'm running version 2.6.11 on Ubuntu 18.04 LTS. I'm thinking of just installing the newer version of smokeping on a new server - Ubuntu 22.04 LTS then just migrate the necessary files across. Any documentation available I can refer to or any help will be really appreciated

New Ubuntu 22.04 is running following for some other application and would like to use the same for Smokeping especially nginx instead of Apache:

- perl v5.34.0
- nginx 1.18.0

**Comment by usaiat** [comment]
Followed this https://sleeplessbeastie.eu/2020/06/19/how-to-install-smokeping/ and it install version 2.7.3 on Ubuntu 22.04 LTS. Copied the Target file and RRD files over from the Ubuntu server hosting version 2.6.11.

Can't seem to start smokeping services. Any help will be really appreciated

#  smokeping --debug
### parsing dig output...OK
Curl: Note: your curl doesn't support the interface feature (option --interface), disabling it
Curl: Note: your curl doesn't support the timeout feature (option -m), disabling it
ERROR: output of '/usr/bin/ssh-keyscan -t dsa,rsa,rsa1 127.0.0.1' does not match (?^i:^# \S+ SSH-)
 at (eval 79) line 1.

#  systemctl reload smokeping
smokeping.service is not active, cannot reload.

**Comment by oetiker** [comment]
Hi @usaiat,

Can you run `/usr/bin/ssh-keyscan -t dsa,rsa,rsa1 127.0.0.1`? What is the output?

**Reply by usaiat** [reply]
Hi @oetiker 
Got this output

Unknown key type "rsa1"

**Comment by vrtareg** [comment]
I have same kind of problem.

Smokeping service does not start on TrueNAS Core jail startup but it starts with "service - R" or "service smokeping start/restart" command.

Enabled debug in rc.conf and can see that smokeping_start is fired up
